guides/common/assembly_administering-hosts.adoc

```diff
@@ -18,8 +18,12 @@ include::modules/proc_creating-a-host-group.adoc[leveloffset=+1]
 
 include::modules/proc_creating-a-host-group-for-each-lifecycle-environment.adoc[leveloffset=+1]
 
+include::modules/proc_adding-a-host-to-the-host-group.adoc[leveloffset=+1]
+
 include::modules/proc_changing-the-host-group-of-a-host.adoc[leveloffset=+1]
 
+include::modules/proc_adding-a-host-to-host-collections.adoc[leveloffset=+1]
```

Comment:
```suggestion
ifdef::katello,satellite,orcharhino[]
include::modules/proc_adding-a-host-to-a-host-collection.adoc[leveloffset=+1]
endif::[]
```
Host collections are available only in products that include Katello.
(We've discussed this with @ekohl in [#1711](https://github.com/theforeman/foreman-documentation/pull/1711/files#r999411719))
Also pls unify the wording of the filename and ID.

Reply:
> 

good to know